scaffold/package/index.js

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env ./node_modules/.bin/babel-node
+
+
+import prompt from 'prompt';
+import {parse} from 'nomnom';
+
+
+const schema = {
+  properties: {
+    name: {
+      description: 'Package name: ',
+      pattern: /^[a-z]+[a-z\-_]+$/,
+      message: 'Name must be only letters, numbers, dashes and underscores',
+      required: true
+    }
+  }
+};
+prompt.message = '>'.green;
```

Comment:
You could bundle these together in an object literal with `Object.assign(prompt, { <insert props here> })`...